rules-tests/CodeQuality/Rector/Class_/CompleteDynamicPropertiesRector/FixtureUnionTypes/multiple_types.php.inc

```diff
@@ -23,7 +23,7 @@ namespace Rector\Tests\CodeQuality\Rector\Class_\CompleteDynamicPropertiesRector
 class MultipleTypes
 {
     /**
-     * @var int|string|bool
+     * @var bool|int|string
```

Comment:
this is sorted docblock by phpstan, previous clone and map cause overlapped types and not stable definition.